pkg/currency/service.go

```diff
@@ -55,3 +56,29 @@ func (service *CurrencyService) ToEur(value float64, from string) (float64, erro
 
 	return value / exchangeRate, nil
 }
+
+func (service *CurrencyService) ToCurrency(to string, value float64, from string) (float64, error) {
+	if from == to {
+		return value, nil
+	}
+
+	exchangeRateString, err := service.redis.HGet(ExchangeRateDataKey, to)
```

Comment:
I feel like getting the exchange rate as float64 could be unified with the one in `ToEur`